_additional/fakeClient/lib/FileReader.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const _ = require('lodash');
+
+module.exports = {
+	/**
+	 * Reads file with provided name and returns its content as an array of file's lines.
+	 * @param {string} filename Name of file.
+	 * @returns {string[]} Lines from file.
+	 */
+	read: function (filename) {
```

Comment:
arrow func

Reply:
Sprawdzałeś jak ta funkcja wygląda w stack trace? Pojawia się i ma nazwę? Jeżeli nie to nie lepiej dać function read(filename), albo zdefiniować funkcję nad i podpiąć niżej?

Reply:
Zdefiniowałem i podpiąłem - w sumie jest czytelniej (szczególnie przy tylko jednej funkcji).